addons/mail/models/mail_alias.py

```diff
@@ -150,11 +150,34 @@ def _clean_and_check_unique(self, name):
 
         catchall_alias = self.env['ir.config_parameter'].sudo().get_param('mail.catchall.alias')
         bounce_alias = self.env['ir.config_parameter'].sudo().get_param('mail.bounce.alias')
+        alias_domain = self._default_alias_domain()
+        new_alias_name = f'{sanitized_name}@{alias_domain}' if alias_domain else sanitized_name
+
+        if sanitized_name in [catchall_alias, bounce_alias]:
+            raise UserError(_('The e-mail alias %(new_alias_name)s is already used as catchall/bounce alias. Please choose another alias.',
```

Comment:
Maybe "is already used as %(alias_duplicate)" where alias_duplicate is either "catchall" or "bounce" to be more precise ?